proto/dapr.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
```

Comment:
Rather than importing the protos, can we use the ones from the `dapr/dapr` repo? I'm very worried about skew between these versions.

Reply:
We may need extract those protos from `dapr/dapr` to a new repo, maybe `dapr/proto`, and we can use it with `git submodule`

Reply:
We may need extract those protos from `dapr/dapr` to a new repo, maybe `dapr/proto`, and we can use it with `git submodule`

Reply:
Yes I think a git submodule would be best. Going to the internet for compiling is something I would like to avoid, though with caching we could make it work...